phpcs.xml

```diff
@@ -16,5 +16,14 @@
     <file>test</file>
 
     <!-- Include all rules from the Laminas Coding Standard -->
-    <rule ref="LaminasCodingStandard"/>
+    <rule ref="LaminasCodingStandard">
+        <!-- Sniff cannot handle multiline array-shape parameter declarations -->
+        <exclude name="Squiz.Commenting.FunctionComment.MissingParamName"/>
```

Comment:
I'd love these kind of checks going away from the whole codestyle config.
Due to the usage of psalm in addition with bumping PHP versions and modernizing code to have native types, especially checks like these are obsolete.